server/routes/shared/riskPresenter.ts

```diff
@@ -0,0 +1,26 @@
+import RiskSummary from '../../models/assessRisksAndNeeds/riskSummary'
+
+export interface RoshAnalysisTableRow {
+  riskTo: string
+  riskScore: string
+}
+
+export default class RiskPresenter {
+  constructor(private readonly riskSummary: RiskSummary | null) {}
+
+  readonly riskSummaryEnabled = this.riskSummary !== null
+
+  get roshAnalysisHeaders(): string[] {
+    return ['Risk to', 'Risk in community']
+  }
+
+  get roshAnalysisRows(): RoshAnalysisTableRow[] {
+    if (this.riskSummary === null) return []
+
+    return Object.entries(this.riskSummary.riskInCommunity).flatMap(([riskScore, riskGroups]) => {
+      return riskGroups.map(riskTo => {
+        return { riskTo, riskScore }
+      })
+    })
+  }
+}
```

Comment:
Do you have tests being performed on these lines of code? I tried searching but couldn't find anything.

Reply:
no but there should be. i'll add something now.